tgui/packages/tgui/interfaces/Safe.js

```diff
@@ -0,0 +1,119 @@
+import { useBackend } from "../backend";
+import { Window } from "../layouts";
+import { Box, Button, Flex, Section } from "../components";
+const safeDialSrc = require('../assets/safe_dial.png');
+
+export const SafeContents = (properties, context) => {
+  const { data, act } = useBackend(context);
+  const {
+    contents,
+  } = data;
+
+  const close = () => {
+    act('open');
+  };
+
+  return (
+    <Section>
+      <Button content="Close" onClick={close} />
+      <Section>
+        {contents.map((content, i) => {
+          const {
+            name, index, sprite,
+          } = content;
+          const retrieve = () => {
+            act('retrieve', { index });
+          };
+          const src = `${sprite}.png`;
+          return (
+            <Flex alignItems="center" key={i}>
+              <img src={src} />
+              <Button content={name} onClick={retrieve} />
+            </Flex>
+          );
+        })}
+      </Section>
+    </Section>
+  );
+};
+
+export const SafeLock = (properties, context) => {
+  const { data, act } = useBackend(context);
+  const {
+    dial, rotation, locked,
+  } = data;
+  const open = () => {
+    act('open');
+  };
+  const turn = (direction, amount) => {
+    act('turn', { amount, direction });
+  };
+
+  return (
+    <Section>
+
+      <Flex alignItems="stretch" flexDirection="column">
+
+        <Flex alignItems="center" flexDirection="column">
+          <Button content="Open" onClick={open} />
+
+        </Flex>
+
+        <Flex justifyContent="space-between" alignItems="center">
```

Comment:
Can you force set the width of the left and right columns to avoid this
![image](https://user-images.githubusercontent.com/25063394/97157932-aeb7f700-1770-11eb-99dd-2f56b67a483d.png)
Yes the buttons on the left are smaller than the ones on the right, but the box being off-centre looks real off